addons/event/views/event_event_views.xml

```diff
@@ -20,6 +20,9 @@
                         context="{'default_event_id': id}">
                         Registration Desk
                     </button>
+                    <div class="alert alert-warning mb-0 ms-2" role="alert">
+                        This event is supposed to be online but no URL address is set.
+                    </div>
```

Comment:
![image](https://github.com/user-attachments/assets/1fc1c822-cbb9-483a-9c5f-f6bdf5a2bc0d)
it looks like extra buttons appear... Could we put it below that row with buttons instead?

Reply:
Having event without website_event is a very specific case therefore we can accept to don't have an alert (seen with SAVP)